src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,55 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
-    public String getSalaryInfo(String[] names, String[] data, String dateFrom, String dateTo) {
-        return null;
+    private static final int DATE_STR_INDEX = 0;
+    private static final int EMPLOYEE_NAME_INDEX = 1;
+    private static final int HOURS_WORKED_INDEX = 2;
+    private static final int INCOME_PER_HOUR_INDEX = 3;
+    private static final DateTimeFormatter date_formatter =
```

Comment:
bad naming for a constant

Reply:
date_formatter =bad naming for a constant
github preview for a comment can be confusing, always look at the latest line of preview code block,I was commenting line 11